### PR TITLE
CFE-2832: Fixed segfault when giving invalid arguments to cf-net

### PR DESCRIPTION
**Pull request by olehermanse**

### CODE REVIEW

**Comment by olehermanse** [comment]
```
$ /var/cfengine/bin/cf-net -H
/var/cfengine/bin/cf-net: option requires an argument -- 'H'
$ /var/cfengine/bin/cf-net -H a.a.a
Command missing, use cf-net --help for more info.
$ /var/cfengine/bin/cf-net stat
Error: no host name (and no policy_server.dat)
$
```

**Comment by olehermanse** [comment]
@cf-bottom Could you trigger a jenkins build for this?

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com/job/pr-pipeline/721/)

https://ci.cfengine.com/job/pr-pipeline/721/

File: cf-net/cf-net.c [hunk 1]
```diff
@@ -313,8 +313,9 @@ static int CFNetParse(int argc, char **argv,
             }
             default:
             {
-                printf("Default optarg = '%s', c = '%c' = %i\n",
-                       optarg, c, (int)c);
+                // printf("Default optarg = '%s', c = '%c' = %i\n",
+                //        optarg, c, (int)c);
```
Comment (vpodzime) [inline cf-net/cf-net.c]: I think we can get rid of this nonsense.

**Pull request merged**